ontology/yaml/resources/SAFETY/entity_types/FSS.yaml

```diff
@@ -23,3 +23,14 @@ FSS_STD:
   - FSS
   uses:
   - fire_alarm
+
+FSS_FA2X_GS:
+  guid: "637223a2-1c85-4ba3-b28f-ba5c96ffce9d"
+  description: "Fire Suppression System with with 2-level alarm and gas suppression."
+  is_canonical: true
+  implements:
+  - FSS
+  - FA2X
+  uses:
+  - failed_panel_alarm
```

Comment:
I would need help to rename this point because it is a general alarm, a catch-all alarm if there is something wrong other than fire or power fail. the description for this was supervisor alarm, in which when it's on a supervisor for the device could check things out. my few suggestions could be:
- event_alarm
- maintenance_alarm
- master_alarm
- process_alarm
- supervisor_alarm
these are not in subfields yet:
- general_alarm
- device_alarm

any more suggestions ma'am?

Reply:
I suggest using `master_alarm` to model this catch-all alarm

Reply:
Noted ma'am. Replaced failed panel alarm to master alarm now